session_rpcserver.go

```diff
@@ -315,9 +315,26 @@ func (s *sessionRpcServer) AddSession(_ context.Context,
 		}
 	}
 
+	// Reserve Session Key & ID.
+	id, localPrivKey, err := s.db.ReserveNewSessionID()
```

Comment:
We could just hold a session mutex in the session RPC server to avoid needing another reserve construct. That way only one session could be created at a time.